css/style.css

```diff
@@ -507,33 +500,86 @@ address {
-.fl-theme-lgdg .floe .floe-image-preferenceIcons { background: url("../images/lgdg-floe-04.png") no-repeat 0 0 / 325px 325px; }
-.fl-theme-lgdg .floe .floe-image-videoPlayer     { background: url("../images/lgdg-floe-03.png") no-repeat 0 0 / 325px 325px; }
+.fl-theme-bw .floe .floe-image-UIO             { background-image: url("../images/bw-floe-02.png"); }
+.fl-theme-bw .floe .floe-image-exploreTool     { background-image: url("../images/bw-floe-01.png"); }
+.fl-theme-bw .floe .floe-image-cloud           { background-image: url("../images/bw-floe-05.png"); }
+.fl-theme-bw .floe .floe-image-preferenceIcons { background-image: url("../images/bw-floe-04.png"); }
+.fl-theme-bw .floe .floe-image-videoPlayer     { background-image: url("../images/bw-floe-03.png"); }
+
+.fl-theme-wb .floe .floe-image-UIO             { background-image: url("../images/wb-floe-02.png"); }
+.fl-theme-wb .floe .floe-image-exploreTool     { background-image: url("../images/wb-floe-01.png"); }
+.fl-theme-wb .floe .floe-image-cloud           { background-image: url("../images/wb-floe-05.png"); }
+.fl-theme-wb .floe .floe-image-preferenceIcons { background-image: url("../images/wb-floe-04.png"); }
+.fl-theme-wb .floe .floe-image-videoPlayer     { background-image: url("../images/wb-floe-03.png"); }
+
+.fl-theme-by .floe .floe-image-UIO             { background-image: url("../images/by-floe-02.png"); }
+.fl-theme-by .floe .floe-image-exploreTool     { background-image: url("../images/by-floe-01.png"); }
+.fl-theme-by .floe .floe-image-cloud           { background-image: url("../images/by-floe-05.png"); }
+.fl-theme-by .floe .floe-image-preferenceIcons { background-image: url("../images/by-floe-04.png"); }
+.fl-theme-by .floe .floe-image-videoPlayer     { background-image: url("../images/by-floe-03.png"); }
+
+.fl-theme-yb .floe .floe-image-UIO             { background-image: url("../images/yb-floe-02.png"); }
+.fl-theme-yb .floe .floe-image-exploreTool     { background-image: url("../images/yb-floe-01.png"); }
+.fl-theme-yb .floe .floe-image-cloud           { background-image: url("../images/yb-floe-05.png"); }
+.fl-theme-yb .floe .floe-image-preferenceIcons { background-image: url("../images/yb-floe-04.png"); }
+.fl-theme-yb .floe .floe-image-videoPlayer     { background-image: url("../images/yb-floe-03.png"); }
+
+.fl-theme-lgdg .floe .floe-image-UIO             { background-image: url("../images/lgdg-floe-02.png"); }
+.fl-theme-lgdg .floe .floe-image-exploreTool     { background-image: url("../images/lgdg-floe-01.png"); }
+.fl-theme-lgdg .floe .floe-image-cloud           { background-image: url("../images/lgdg-floe-05.png"); }
+.fl-theme-lgdg .floe .floe-image-preferenceIcons { background-image: url("../images/lgdg-floe-04.png"); }
+.fl-theme-lgdg .floe .floe-image-videoPlayer     { background-image: url("../images/lgdg-floe-03.png"); }
+
+.fl-theme-bw .floe .hewlett-logo-container svg {
+    border: 1px solid #000;
```

Comment:
You might be able to simplify all of the contrast themes by using [currentColor](https://developer.mozilla.org/en/docs/Web/CSS/color_value#currentcolor_keyword). You'd then be able to have the same declaration for all of the themes.